src/frontend/js/components/ReferralActivityIndicator/ActivityCloseReferral.tsx

```diff
@@ -0,0 +1,34 @@
+import { DateTime, Duration } from 'luxon';
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { GenericErrorMessage } from 'components/GenericErrorMessage';
+import { Spinner } from 'components/Spinner';
+import { useReferral } from 'data';
+import { Referral, ReferralActivityClosed } from 'types';
+
+const messages = defineMessages({
+  loadingReferral: {
+    defaultMessage: 'Loading referral...',
+    description:
+      'Accessible text for the spinner while loading the referral in the activity indicator for urgency change.',
+    id:
+      'components.ReferralActivityIndicator.ActivityCloseReferral.loadingReferral',
+  },
+});
+
+interface ActivityCloseReferralProps {
+  activity: ReferralActivityClosed;
+}
+
+export const ActivityCloseReferral: React.FC<ActivityCloseReferralProps> = ({
+  activity,
+}) => {
+  return (
```

Comment:
Don't need a return statement here, you can just remove the {} around the function body